admin_guide/ipsec.adoc

```diff
@@ -168,6 +168,12 @@ conn clear
 	priority=100
 ----
 <1> Replace <cert_nickname> with the certificate nickname from step one.
++
+[IMPORTANT]
+====
+The Amazon/Azure internal cloud network does not route IPsec `ESP` or `AH` packets. These packets need to be encapsulated in `UDP`. While normally the NAT detection takes care of this `ESP` in `UDP` encapsulation, if NAT is not detected, you can force encapsulation by setting `encapsulation=yes`.
```

Comment:
I'm not sure where you'd add `encapsulation=yes`. I think it might be clearer if you put that line into the required section of the code sample and added a separate annotation instead of a note, sort of like this:

```
encapsulation=yes <2>
```

```
<2> If you do not use NAT, you must include `encapsulation=yes` in the configuration to force encapsulation. The Amazon and Azure internal cloud networks do not route IPsec `ESP` or `AH` packets. These packets must be encapsulated in `UDP`, and if it is configured, NAT detection configures the `ESP` in `UDP` encapsulation. If you use NAT, omit this parameter and value.
```

Reply:
OK. I've changed that in the 2 sections.